samples/subsys/usb/uvc/README.rst

```diff
@@ -0,0 +1,185 @@
+.. zephyr:code-sample:: uvc
+   :name: USB Video sample
```

Comment:
"sample" is not to be used for sample names (everything is a sample!)

```suggestion
   :name: USB Video
```

Reply:
feel free to be creative tho :) "USB Video" is maybe on the short side, so I wonder if "USB Video webcam" would work better? Probably?

Reply:
Attempt with "USB Video, webcam".

Reply:
not sure about the comma tbh but whatever :)
Let's wait for CI to re-build the docs but virtually +1 from me :)